javascript-loops-practice/includes-seven.js

```diff
@@ -0,0 +1,12 @@
+/* exported includesSeven */
+function includesSeven(array) {
+  var x = false;
+  for (i = 0; i < array.length; i++) {
+    if (array[i] === 7) {
+      x = true;
+    } else {
+      x = false;
+    }
```

Comment:
Try returning immediately in the loop if `7` is found. That should eliminate your need for an `else` block.